cmd/release-controller/sync_analysis.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	imagev1 "github.com/openshift/api/image/v1"
+	"k8s.io/klog"
+)
+
+func (c *Controller) ensureAnalysisJobs(release *Release, releaseTag *imagev1.TagReference) error {
```

Comment:
we spoke about making this only create analysis jobs one out of 10 runs.  If you want to create that logic, now is the time.

I would use an atomic int and a mod